src/cascadia/TerminalSettingsEditor/Utils.cpp

```diff
@@ -102,4 +102,11 @@ namespace winrt::Microsoft::Terminal::Settings
 
         return L"";
     }
+
+    hstring LocalizedNameForEnumName(const std::wstring_view sectionAndEnumType, const std::wstring_view enumValue, const std::wstring_view propertyType)

```

Comment:
When I add a new enum value, or just add a new enum entirely, what do I need to do to make sure this won't break? Will this silently crash when I add a new enum value _unless_ I also add the correct resource to `Resources.resw`?

Reply:
pretty much. it should be instant and obvious when you open the settings UI tho

Reply:
yup at least for the SUI, you'll have to make sure you line up your json enum strings in `JSON_ENUM_MAPPER` with the resource key, it was kinda hard to make a helper like this without assuming some sort of naming convention in `resources.resw`